js/utility/bc-google-analytics.js

```diff
@@ -90,8 +93,8 @@ bcpl.utility.googleAnalytics = (() => {
 		addOutboundLinkTracking();
 	};
 
-	const getDefaultKey = (action) => hasOwnProperty(defaultGoogleEvents, action) ? 
-		defaultGoogleEvents[action] : 
+	const getDefaultKey = (action) => hasOwnProperty(defaultGoogleEvents, action) ?
+		defaultGoogleEvents[action] :
 		null;
```

Comment:
Can you change the operator placement to match what's on lines 85-87 and 89-91?

Reply:
done